### PR TITLE
Initial commit for azure-cosmos 3.0.2 recipe

### PR DESCRIPTION
**Pull request by yehoshuadimarsky**

### CODE REVIEW

**Comment by yehoshuadimarsky** [comment]
Hey, can anyone please help me figure out why these builds are failing? This is my first time doing this... The source code in the recipe is super simple, just an SDK built around an API and only uses `requests`, so I thought would be very straightforward.

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,83 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (xhochy) [inline recipes/example/meta.yaml]: Please don't delete the example

File: recipes/azure-cosmos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 081b3f592c3cf88c4d423fbb8184732cae0427cc5188cfed4835d9cdb64fa5f7
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/azure-cosmos/meta.yaml]: These are host dependencies, not build.
```suggestion
  host:
```

Comment (yehoshuadimarsky) [reply recipes/azure-cosmos/meta.yaml]: Thanks!! That was quick.

Comment (yehoshuadimarsky) [reply recipes/azure-cosmos/meta.yaml]: Shoot, the build still fails, even though I included `setuptools` in the recipe. @xhochy any more ideas?

File: recipes/azure-cosmos/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - pip
+  run:
```
Comment (xhochy) [inline recipes/azure-cosmos/meta.yaml]: `setuptools` must also be listed here as it is a runtime dependency to declare the namespace.

**Pull request merged**